samples/cpp/hello_reshape_ssd/main.cpp

```diff
@@ -53,7 +53,7 @@ int main(int argc, char* argv[]) {
         // try to find it.
         ov::NodeVector ops = model->get_ops();
         auto it = std::find_if(ops.begin(), ops.end(), [](std::shared_ptr<ov::Node> node) {
-            return node->get_type_info().name == ngraph::op::DetectionOutput::get_type_info_static().name;
+            return node->get_type_info() == ngraph::op::DetectionOutput::get_type_info_static();
```

Comment:
why this change is needed? Is it not BW compatible change?

Reply:
The original code was incorrect. We cannot guarantee that we use the same pointers.

Reply:
Where do we have pointers?